src/utils/replayer/replayer.cpp

```diff
@@ -0,0 +1,226 @@
+#include <Corrade/Containers/GrowableArray.h>
+#include <Corrade/Utility/Arguments.h>
+#include <Corrade/Utility/DebugStl.h>
+#include <Corrade/Utility/Json.h>
+#include <Corrade/Utility/Path.h>
+#include <Magnum/DebugTools/FrameProfiler.h>
+#include <Magnum/GL/DefaultFramebuffer.h>
+#include <Magnum/Platform/GlfwApplication.h>
+
+#include "esp/sensor/CameraSensor.h"
+#include "esp/sim/ReplayBatchRenderer.h"
+
+namespace {
```

Comment:
How about some file-level/class-level documentation about this Replayer app? Even just a sentence describing what it is.

Reply:
Oh yeah, definitely, kinda forgot to do this when the code transformed from a rough test to an actual utility :D

Reply:
Done in https://github.com/facebookresearch/habitat-sim/pull/1947/commits/14a81f71f2a90b2d85b72b4a9edd75ce83a2a026 -- it's a `--help` text, similarly to the `viewer` app.